clang/test/CodeGenSYCL/mark-kernel-name.cpp

```diff
@@ -0,0 +1,24 @@
+// RUN: %clang_cc1 -triple x86_64-linux-pc  -fsycl-is-host -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang_cc1 -triple x86_64-linux-pc  -fsycl-is-device -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+
```

Comment:
Could you please add a comment describing the test, and use the mock SYCL headers as per contributing guidelines here - https://github.com/intel/llvm/blob/sycl/CONTRIBUTING.md